test/verify/files/mock-insights

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/python3
```

Comment:
Curious, how does that work on RHEL 7?

Reply:
Good question!

Reply:
Our `rhel-7-8` image has Python 3 installed.

Reply:
Our rhel-7-8 image has Python 3 installed.